### ISSUE TITLE
Downloading error

### ISSUE THREAD

**Issue by vosi** [flexiondotorg/oab-java6#55]
Downloading jdk-7u6-linux-i586.tar.gz :    http://: Invalid host name.
Downloading jdk-6u34-linux-i586.bin :    http://: Invalid host name.

**Comment by thearc77** [comment]
i get the same:

[x] Getting previous releases download page   --2012-08-30 16:46:32--  http://www.orac$
Resolviendo www.oracle.com (www.oracle.com)... 184.24.65.55
Conectando con www.oracle.com (www.oracle.com)[184.24.65.55]:80... conectado.
Petición HTTP enviada, esperando respuesta... 200 OK
Longitud: no especificado [text/html]
Grabando a: “/tmp/oab-download.html”

```
 0K .......... .......... .......... .......... ..........  125K
50K .......... .......... .......... .......... ..........  371K
```

   100K .......... .......... .......... .......... ..........  373K
   150K .......... .......... .......... .......... ..........  289K
   200K .......... .......... .......... .......... ..........  366K
   250K .......... .......... ........                          209K=1,1s

2012-08-30 16:46:34 (250 KB/s) - “/tmp/oab-download.html” guardado [285465]

^H^H16821's retcode: 0
success
 [x] Downloading jdk-6u34-linux-i586.bin :    http://: Nombre de equipo inválido.
^H^H16849's retcode: 1

**Comment by f0t0n** [comment]
Same issue:

```

root@lotus:~# ./oab-java.sh -7
oab-java.sh v0.2.5 - Create a local 'apt' repository for Sun Java 6 and/or Oracle Java 7 packages.

Copyright (c) Martin Wimpress, http://flexion.org. MIT License

By running this script to download Java you acknowledge that you have
read and accepted the terms of the Oracle end user license agreement.

* http://www.oracle.com/technetwork/java/javase/terms/license/

If you want to see what this is script is doing while it is running then execute
the following from another shell:

  tail -f /root/oab-java.sh.log

 [x] Installing Java build requirements success
 [x] Making build directories success
 [x] Removing clones of https://github.com/rraptorr/oracle-java7 success
 [x] Cloning https://github.com/rraptorr/oracle-java7 success
 [x] Checking out v7.6-1 success
 [x] Getting Java SE download page success
 [x] Getting current release download page success
 [x] Getting previous releases download page success
 [x] Downloading jdk-7u6-linux-i586.tar.gz :  failed
 [i] Showing the last 5 lines from the logfile (/root/oab-java.sh.log)...
1587's retcode: 0
success
 [x] Downloading jdk-7u6-linux-i586.tar.gz :    http://: Invalid host name.
1608's retcode: 1
failed
root@lotus:~# 
```

**Comment by svein** [comment]
Same error with jdk-7u7

./oab-java.sh -7
oab-java.sh v0.2.5 - Create a local 'apt' repository for Sun Java 6 and/or Oracle Java 7 packages.

Copyright (c) Martin Wimpress, http://flexion.org. MIT License

By running this script to download Java you acknowledge that you have
read and accepted the terms of the Oracle end user license agreement.
- http://www.oracle.com/technetwork/java/javase/terms/license/

If you want to see what this is script is doing while it is running then execute
the following from another shell:

  tail -f /home/user/oab-java6/oab-java.sh.log

 [x] Installing Java build requirements success
 [x] Making build directories success
 [x] Removing clones of https://github.com/rraptorr/oracle-java7 success
 [x] Cloning https://github.com/rraptorr/oracle-java7 success
 [x] Checking out v7.7-1 success
 [x] Getting Java SE download page success
 [x] Getting current release download page success
 [x] Getting previous releases download page success
 [x] Downloading jdk-7u7-linux-i586.tar.gz :  failed
 [i] Showing the last 5 lines from the logfile (/home/user/oab-java6/oab-java.sh.log)...
22647's retcode: 0
success
 [x] Downloading jdk-7u7-linux-i586.tar.gz :    http://: Invalid host name.
22664's retcode: 1
failed

**Comment by mikkorantalainen** [comment]
I'm getting the same issue with 7u7. In addition, the script does not catch the failure but says "success". Resulting files in /usr/local/oab/pkg are zero bytes long and building the pagkages will obviously fail. I'm running Ubuntu 10.04 LTS 64 bit and I have wget version 1.12 in case that makes the difference.

**Comment by GiovanniES** [comment]
Well, I know this is not a solution to the problem, but at least I could get the script to finish:

I looked for jdk and found it in http://www.oracle.com/technetwork/java/javase/downloads/jdk7u7-downloads-1836413.html. I agreed to the license fro Oracle and download both files jdk-7u7-linux-i586.tar.gz and jdk-7u7-linux-x64.tar.gz. I also looked for and download the Java Cryptography Extensions from http://www.oracle.com/technetwork/java/javase/downloads/jce-7-download-432124.html. Then, I moved the three files to /var/local/oab/pkg.

Then, I comment the lines 474-475 and 502-503 (I want java7) to avoid trying the automatic downloading. The rest of the script went smoothly when I ran sudo ./oab-java.sh -7. I hope this comment will help those who do need to upgrade right now.
